src/status_im/react_native/resources.cljs

```diff
@@ -46,6 +46,11 @@
    :collectible        (js/require "../resources/images/ui/collectible.png")
    :collectible-dark   (js/require "../resources/images/ui/collectible-dark.png")
    :hand-wave          (js/require "../resources/images/ui/hand-wave.png")
+   :zksync             (js/require "../resources/images/ui/zksync.png")
```

Comment:
could you elaborate why we need these images ?

Reply:
these are wallet icons
![Screenshot 2022-07-21 at 5 26 09 PM](https://user-images.githubusercontent.com/64726664/180208172-f15e9f46-7d10-40dc-b323-8cd4e336c070.png)

Reply:
oh its just hardcoded images like and example of coins?

Reply:
yes just so that it looks good with mock data for now

Reply:
Hope I can mark this as resolved now